api/Request.json

```diff
@@ -1054,10 +1054,10 @@
               "version_added": "15"
             },
             "firefox": {
-              "version_added": null
+              "version_added": false
```

Comment:
Seems right, `keepalive` support is tracked by https://bugzilla.mozilla.org/show_bug.cgi?id=1342484